packages/console/src/hooks/use-connector-groups.ts

```diff
@@ -17,10 +18,19 @@ const useConnectorGroups = () => {
     return getConnectorGroups(data);
   }, [data]);
 
+  const hasSupportNativePlatformTarget = useMemo(
+    () =>
+      groups?.some(({ connectors }) =>
+        connectors.some(({ target }) => supportNativePlatformTargets.includes(target))
```

Comment:
no need to use two nested loop? since connectors are grouped by `target`.

Reply:
updated.